src/PostgREST/Error.hs

```diff
@@ -587,6 +629,8 @@ data ErrorCode
   | JWTErrorCode02
   -- Internal errors related to the Hasql library
   | InternalErrorCode00
+  -- JSON parse error, related to invalid json sent via RAISE for custom response
+  | JSONParseErrorCode00
```

Comment:
There's a pretty similar error code for an incorrect format on `set_config('response.headers',..`, see:

- https://postgrest.org/en/stable/references/transactions.html#guc-resp-hdrs
- https://postgrest.org/en/stable/references/errors.html#pgrst111

So this error should belong to the same group, likely a `PGRST121`.